tests/resources/Docker-Util.robot

```diff
@@ -201,4 +201,86 @@ Verify Volume Inspect Info
     Should Be Equal As Integers  ${rc}  0
 
     :FOR  ${item}  IN  @{checkList}
-    \   Should Contain  ${mountInfo}  ${item}
\ No newline at end of file
+    \   Should Contain  ${mountInfo}  ${item}
+
+Log All Containers
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a
+    Run Keyword If  ${rc} != 0  Log To Console  Remaining containers - ${out}
+
+Does Images Exist
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images -q
+    ${len}=  Get Length  ${output}
+    Return From Keyword If  ${len} == 0  ${false}
+    [Return]  ${true}
+
+Does Containers Exist
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a -q
+    ${len}=  Get Length  ${output}
+    Return From Keyword If  ${len} == 0  ${false}
+    [Return]  ${true}
+
+Does Volumes Exist
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume ls -q
+    ${len}=  Get Length  ${output}
+    Return From Keyword If  ${len} == 0  ${false}
+    [Return]  ${true}
+
+Does Networks Exist
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} networks ls -q
+    ${len}=  Get Length  ${output}
+    Return From Keyword If  ${len} == 0  ${false}
+    [Return]  ${true}
+
+Remove All Containers
+    ${exist}=  Does Containers Exist
+    Run Keyword If  ${exist}  Log To Console  Stopping and removing all containers from %{VCH-NAME}
+    Return From Keyword If  ${exist} == ${false}  0
+
+    Run Keyword If  ${exist}  Kill All Containers
+    Run Keyword If  ${exist}  Run  docker %{VCH-PARAMS} rm $(docker %{VCH-PARAMS} ps -a -q)
```

Comment:
Should we simply `rm -f` containers here, just to be sure? That way we can skip the `kill` step too.